enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/dependency/DependencyConvergence.java

```diff
@@ -142,7 +146,7 @@ private String buildConvergenceErrorMsg(List<DependencyNode> nodeList) {
     @Override
     public String toString() {
         return String.format(
-                "DependencyConvergence[includes=%s, excludes=%s, uniqueVersions=%b]",
-                includes, excludes, uniqueVersions);
+                "DependencyConvergence[includes=%s, excludes=%s, uniqueVersions=%b, scopes=%s]",
```

Comment:
no need to include the constant list scopes in the output here

Reply:
It is configurable, thus not static (see my answer to the next comment).